### PR TITLE
completed javascript-primitives-and-variables

### PR DESCRIPTION
**Pull request by jdfenlason**
Screen Shot 2021-03-15 at 1 53 57 PM

### CODE REVIEW

File: javascript-primitives-and-variables/main.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+var fullName = 'Jake Fenlason';
+
+var isCool = 'True';
```
Comment (codingcodymiller) [inline javascript-primitives-and-variables/main.js:3]: This should be a boolean

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great start! Also, please include your console logs in the code for this exercise.

**Comment by jdfenlason** [comment]
Screen Shot 2021-03-15 at 2 04 35 PM

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey man! Please include your `console.log` calls in the js file itself

**Comment by jdfenlason** [comment]
Screen Shot 2021-03-16 at 3 59 34 PM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**